getting_started/build-guide/build-from-scratch.rst

```diff
@@ -125,97 +129,116 @@ parameter.
    If the gitcompile script does not work, refer to the INSTALL file for
    manual build instructions.
 
-Step 2 Build toolchain from source
-==================================
+Create or append to the ``LD_LIBRARY_PATH`` environment variable.
+
+.. code-block:: bash
+
+   export LD_LIBRARY_PATH=~/work/sof/alsa-lib/src/.libs:$LD_LIBRARY_PATH
 
-Build cross-compiler
---------------------
+Step 2 Build toolchains from source
+===================================
 
-Build the xtensa cross compiler with crosstool-ng for Intel |BYT|,
+Build the xtensa cross-compilation toolchains with crosstool-ng for Intel |BYT|,
 |CHT|, |HSW|, |BDW|, |APL|, |CNL| platforms and NXP i.MX8/i.MX8X/i.MX8M
 platforms.
 
-Clone both repos and check out the sof-gcc8.1 branch.
+crosstool-ng
+------------
+
+Clone both repos and check out the ``sof-gcc8.1`` branch.
 
 .. code-block:: bash
 
    cd ~/work/sof/
    git clone https://github.com/thesofproject/xtensa-overlay
+   git clone https://github.com/thesofproject/crosstool-ng
    cd xtensa-overlay
    git checkout sof-gcc8.1
-   cd ~/work/sof/
-   git clone https://github.com/thesofproject/crosstool-ng
-   cd crosstool-ng
+   cd ../crosstool-ng
    git checkout sof-gcc8.1
```

Comment:
we are in gcc9x now
https://github.com/thesofproject/crosstool-ng/tree/sof-gcc9x
@lgirdwood the Docker image is still using gcc8.1 branch like here, do we need to make the update?

Reply:
As this PR is already very large, I want to draw the following line: fixes but no "upgrade". These can be tested and done in a later PR.

Reply:
Is there any way to "inline" or link (better) original scripts from the sof repo, in case the content is duplicated to reduce further maintenance of the copy?

Reply:
@mmaka1 I can add http:// links on the script names mentioned here, that's a good idea thanks. On the other hand I don't understand what duplication you're referring to, example?

Reply:
@mmaka1 I tried https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html and it Just Worked, pretty cool. It doesn't work inside `.. code-block::` but the texts before can be rephrased to add the links. I can do this in a later PR.

Reply:
> @mmaka1 I tried https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html and it Just Worked, pretty cool. It doesn't work inside .. code-block:: but the texts before can be rephrased to add the links. I can do this in a later PR.

Submitted in PR #213